client/pages/recipient-start.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import SpotifyLogin from './spotify-login';
+
+export default function RecipientStart(props) {
+  return (
+    <>
+      <div className="overlay position-absolute"></div>
+      <div className="row align-center flex-column position-absolute padding-3rem desktop-style">
+        <h1 className="font-size-48 no-margin">Ahoy Matey!</h1>
+        <h2 className="font-size-24 text-center">Click on the shell to connect your Spotify account</h2>
+        <SpotifyLogin callback='http://localhost:3000/callback/recipient' />
```

Comment:
You're going to need to make this an environment variable. That means moving it to your `.env` file and including `SPOTIFY_AUTH_CALLBACK` (or whatever you name it) to your environment plugin settings for `webpack.config.js`.